ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1173,21 +1181,23 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
                   You must select a billing account</div>}
                 {errors.primaryPurpose && <div> You must choose at least one primary research
                   purpose (Question 1)</div>}
+                {errors.diseaseOfFocus && <div> You must specify a disease of focus and it should be at most 80 characters</div>}
                 {errors.otherPrimaryPurpose && <div> Other primary purpose should be of at most 500 characters</div>}
                 {errors.anticipatedFindings && <div> Answer for <i>What are the anticipated findings
-                  from the study? (Question # 2.1)</i> cannot be empty</div>}
+                  from the study? (Question 2.1)</i> cannot be empty</div>}
                 {errors.scientificApproach && <div> Answer for <i>What are the scientific
-                  approaches you plan to use for your study (Question # 2.2)</i> cannot be empty</div>}
+                  approaches you plan to use for your study (Question 2.2)</i> cannot be empty</div>}
                 {errors.intendedStudy && <div> Answer for<i>What are the specific scientific question(s) you intend to study
-                  (Question # 2.3)</i> cannot be empty</div>}
-                {errors.specificPopulation && <div> You must specify a population of study</div>}
-                {errors.diseaseOfFocus && <div> You must specify a disease of focus and it should be at most 80 characters</div>}
-                {errors.researchOutcoming && <div> You must specify the outcome of the research</div>}
-                {errors.disseminate && <div> You must specific how you plan to disseminate your research findings</div>}
+                  (Question 2.3)</i> cannot be empty</div>}
+                {errors.disseminate && <div> You must specific how you plan to disseminate your research findings (Question 3)</div>}
```

Comment:
Added (Question 3) and (Question 4) per Karthik's request, to have consistency with labeling there.